sim/druid/item_sets_pve.go

```diff
@@ -677,11 +677,57 @@ var ItemSetGenesisFury = core.NewItemSet(core.ItemSet{
 	Bonuses: map[int32]core.ApplyEffect{
 		// Each time you Dodge while in Dire Bear Form, you gain 10% increased damage on your next Mangle or Swipe, stacking up to 5 times.
 		2: func(agent core.Agent) {
-			// TODO Bear
+			druid := agent.(DruidAgent).GetDruid()
+
+			druid.Tank2PieceAqProcAura = druid.RegisterAura(core.Aura{
+				Label:     "Feral 2P Bonus Proc",
+				ActionID:  core.ActionID{SpellID: 1213188},
+				Duration:  time.Second * 10,
+				MaxStacks: 5,
+				OnStacksChange: func(aura *core.Aura, sim *core.Simulation, oldStacks, newStacks int32) {
+					if newStacks == 0 {
+						druid.MangleBear.DamageMultiplierAdditive -= float64(oldStacks) * .1
+						druid.SwipeBear.DamageMultiplierAdditive -= float64(oldStacks) * .1
+					} else {
+						druid.MangleBear.DamageMultiplierAdditive += .1
+						druid.SwipeBear.DamageMultiplierAdditive += .1
+					}
```

Comment:
This is a bit simpler way to handle stack changes

```suggestion
				druid.MangleBear.DamageMultiplierAdditive += 0.1 * (newStacks - oldStacks)
				druid.SwipeBear.DamageMultiplierAdditive += 0.1 * (newStacks - oldStacks)
```